apps/web/src/components/AssessmentStepTracker/AssessmentResults.tsx

```diff
@@ -88,19 +145,72 @@ const AssessmentResult = ({
         data-h2-gap="base(0 x.25)"
         data-h2-padding="base(x.125 0)"
         data-h2-width="base(100%)"
+        {...(isBookmarked && {
+          "data-h2-radius": "base(5px)",
+          "data-h2-background-color": "base(primary.lightest)",
+        })}
       >
-        <BookmarkIcon
-          {...iconStyles}
-          data-h2-color="base(gray)"
-          data-h2-flex-shrink="base(0)"
-        />
+        {isBookmarked ? (
+          <Button
+            mode="icon_only"
+            color="primary"
+            onClick={toggleBookmark}
+            icon={BookmarkIconSolid}
+            aria-label={intl.formatMessage(
+              {
+                defaultMessage:
+                  "Un-bookmark {candidateName} from top of column.",
```

Comment:
Isn't the verb here meant to be **pin**?

Reply:
I've updated this with [Change some copy to use 'pin' instead of 'bookmark'](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8767/commits/bd5d8656334aaf072525789e776fe3d8e36a5131) but I feels like it loses some consistency since we use "Bookmark" elsewhere.

Reply:
@GCTC-NTGC/designers thoughts?

Reply:
@mnigh @JamesHuf Tough to say what Jerbo's original intent here was, but from my fresh-eyes perspective, it makes more sense to use _pin_ in this context, because they're being literally pinned to the top of the list. I agree though that the icon we've chosen makes this name confusing. Let me chat with Lauren quickly to make sure I'm not modifying something very deliberate before giving you any further direction.

Reply:
@mnigh @JamesHuf So I spoke with Lauren about this, and her direction was that bookmarking/pinning are both fine, but that it's important for us to keep the main requirement in mind, which is that the behaviour should be perceived as agnostic and avoid connotations of rankings/ratings/etc.

That all said, the fact that the change from _bookmark_ to pin feels very last minute (post-PR as opposed to issue hand-off) makes me lean towards keeping this as is (bookmark) until we can have a proper conversation with Jerbo about the direction he wants to take.